pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -855,6 +856,11 @@ def append_tk_command_to_qiskit(
 supported_gate_rebase = AutoRebase(_protected_tket_gates)
 
 
+def _has_implicit_permutation(circ: Circuit) -> bool:
```

Comment:
This would be better defined in pytket rather than this extension: there's nothing specific to qiskit about it. But for now there is no need to factor it out as a method, since it's only used in one place.

Reply:
Agreed. Maybe it should just be a property of `Circuit`?

Reply:
Yes. But for the purpose of this PR the check can just be inlined.